lib/tasks/general.rake

```diff
@@ -19,3 +19,12 @@ task fetch_all_user_bikes: :environment do
     FetchBikeIndexBikesJob.perform_later(user)
   end
 end
+
+task fetch_bikes_by_binx_id: :environment do
+  Bike.find_in_batches(batch_size: 100) do |batch|
+    pp 'Batching all bicycles..'
+    batch.each { |bike| FetchBikeByBikeIndexIdJob.perform_later(bike) }
```

Comment:
Ideally rake tasks will finish quickly (and successfully).

Rather than using `sleep` here, you should use the `wait` functionality [built into activejob](http://edgeguides.rubyonrails.org/active_job_basics.html#enqueue-the-job)